regression/pageObjects/productCategories.js

```diff
@@ -0,0 +1,17 @@
+class productCategories {
+    
+    async clickFashionPlus(){
+        let fashionPlus = element.all(by.xpath("/html[1]/body[1]/div[1]/main[1]/div[2]/div[1]/div[1]/div[1]/div[1]/button[1]"));
```

Comment:
we should avoid writing locators in this format
try this:
`//div/p[@class='parent-name'][text()='Fashion']/following-sibling::button`

More: https://stackoverflow.com/questions/45574159/xpath-how-to-select-elements-that-are-related-to-other-on-the-same-level

Reply:
And of course you can change 'Fashion' to #{list_name} so that you can easily click on every "+" element for Accessories, Electronics etc